mypy/test/data/check-modules.test

```diff
@@ -739,3 +739,18 @@ y = 42
 [builtins fixtures/module.py]
 [out]
 tmp/main.py:4: error: Unsupported left operand type for + ("int")
+
+[case testSuperclassInImportCycle]
```

Comment:
Does this assume that import loops are processed in a particular order? If yes, as we might change the order in the future, it may be better to have another test case would demonstrate the issue if the modules in the import cycle were processed in the opposite order.